### PR TITLE
Add some overview info about TAing

### LINKED ISSUE #27: Add something about how to find the Mailchimp list sign up urls
The way I did it last time was to make a _new signup form_ which seems wrong.

### PR DESCRIPTION
**Pull request by mxie**
This includes a link to our TA interest form and potentially the
TA-selection process and whatever other info in the future.

Closes #27.

What it'd look like when we navigate to the `TAs` folder now:
cookbook 2018-06-08 11-29-02

### CODE REVIEW

File: TAs/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# TAs
+
+Our TAs is a huge part of the workshop experience for our participants. It's
+important that we take great care in both selecting the TAs and preparing them
+for an upcoming workshop.
```
Comment (edwardloveall) [inline TAs/README.md]: What about a note saying what our criteria is on a high level? Diverse across backgrounds, genders, experience, etc.

Comment (mxie) [reply TAs/README.md]: Yeah! How about this: 12333d0c34b6d89d3302e6e2e858eeb47733b222?

**Comment by amandabeiner** [comment]
👍

File: TAs/README.md (hunk 1 above)
Comment (raadler) [reply TAs/README.md]: I spot a typo on line 3. Should be  "Our TA *are* a huge part of the workshop..."

Comment (mxie) [reply TAs/README.md]: Yup got it - thanks!

**Pull request merged**